carbonj.service/src/main/java/com/demandware/core/metric/Util.java

```diff
@@ -24,8 +24,8 @@ public class Util
 
     private static GraphiteReporter graphiteReporter = null;
 
-    public static String getGraphiteMetricPrefix( String graphiteMetricsPrefix, int podId, String groupId,
```

Comment:
Is it possible to add unit test for this ?

Reply:
In order to do this, I'd have to mock [InetAddress.getLocalHost()](https://github.com/salesforce/carbonj/blob/412549233c2a6f3ed30c4915877f2c7fd7a5abf1/carbonj.service/src/main/java/com/demandware/core/metric/Util.java#L31).   Since it's a static method, I believe we'd need to use Powermock.  It looks like Powermock isn't being used currently in this repo, so it would take some time to get this working.

Reply:
I added a unit test